File: DailyFlash Solutions/DailyFlash Solutions/64-DailyFlash_Solutions/03_Apr_Solution_One/Java/Program0.java
import java.io.*;

class Series{


	static int calculateFactorial(int num){

		int fact =1;

		while(num>0){
			fact *=num;
			num -= 1;
		}

		return fact;
	}



	public static void main(String ... nk)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		int a,x,y;
		float res=0;

		while(true){

			try{

				System.out.println("Enter the length of Series : ");
				a = Integer.parseInt(br.readLine());
				System.out.println("Enter X : ");
				x = Integer.parseInt(br.readLine());
				System.out.println("Enter Y : ");
				y = Integer.parseInt(br.readLine());
				break;

			}catch(NumberFormatException nfe){

				System.out.println("NFE : Enter valid number !!!");
			}
		}



		for(int i=1; i<=a; i++)
		res+=1/(Math.pow((x+y),i)*(calculateFactorial(i)));



		System.out.printf("Addition of series is : %.2f\n",res);


	}

}

File: DailyFlash Solutions/DailyFlash Solutions/64-DailyFlash_Solutions/03_Apr_Solution_One/C/Program0.c
#include<stdio.h>
#include<math.h>

double pow(double,double);

int calculateFactorial(int num){

	int fact =1;
	while(num>0){
		fact *=num;
		num -= 1;
	}

	return fact;
}



void main(){

	int a,x,y;
	double res=0;

	printf("Enter the limit : ");
	scanf("%d",&a);
	printf("Enter X: ");
	scanf("%d",&x);
	printf("Enter Y: ");
	scanf("%d",&y);


	for(int i=1; i<=a; i++)
		res+=1/(pow((x+y),i)*(calculateFactorial(i)));
		

	printf("Result of series is %.3lf\n",res);





}
